exercises/F-predicates/exercise.js

```diff
@@ -6,10 +6,21 @@
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {
```

Comment:
This function works for the console log assertions but think how this would work if a negative number was passed that wasn't -5